webpack/assets/javascripts/bundle.js

```diff
@@ -25,22 +25,8 @@ import * as configReportsModalDiff from './foreman_config_reports_modal_diff';
 import * as classEditor from './foreman_class_edit';
 import * as dashboard from './dashboard';
 import * as spice from './spice';
-import './bundle_datatables';
-import './bundle_lodash';
```

Comment:
I may be missing it, where are we making window._ available if the bundle is removed?

Reply:
Lodash put it there automatically once you require it, the `bundle_lodash` just replaced the `window._` with another leaner object.